app/src/views/private/components/header-bar.vue

```diff
@@ -91,12 +91,13 @@ onUnmounted(() => {
 	align-items: center;
 	justify-content: flex-start;
 	width: 100%;
-	height: var(--header-bar-height);
+	height: calc(var(--header-bar-height) + var(--theme--header--border-width));
```

Comment:
Oddly this seems to break when `--theme--header--border-width` is `0`, which is the default value:

![](https://github.com/directus/directus/assets/42867097/b3ca8e5f-2011-4349-8017-01aa40a156bc)

How it looked before this PR:

![](https://github.com/directus/directus/assets/42867097/bd3595ac-026a-44e8-9cdc-815932e4fa38)

Turns out the resulting `calc(61px + 0)` is an invalid operation, or referred as "unitless zero", as described in [the specs](https://drafts.csswg.org/css-values-3/#calc-type-checking):

> Note: Because `<number-token>`s are always interpreted as `<number>`s or `<integer>`s, "unitless 0" `<length>`s aren’t supported in `calc()`. That is, `width: calc(0 + 5px);` is invalid, even though both `width: 0;` and `width: 5px;` are valid. 

With that in mind, updating all instances of `borderWidth: '0'` to `borderWidth: '0px'` in themes packages' light and dark theme seems to resolve it. Updated via commit https://github.com/directus/directus/pull/20226/commits/bc71ef70493aaf30e7dcf85b32fa7701f08e06c3.

I do wonder should we somehow document this for future reference? or if the same gotcha is faced by future themes creators.